easybuild/easyconfigs/d/DIRAC/DIRAC-22.0-foss-2021a.eb

```diff
@@ -0,0 +1,63 @@
+easyblock = 'CMakeMake'
+
+name = 'DIRAC'
+version = '22.0'
+
+homepage = 'http://www.diracprogram.org'
+description = "DIRAC: Program for Atomic and Molecular Direct Iterative Relativistic All-electron Calculations"
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+toolchainopts = {'usempi': True, 'i8': False}
+
+source_urls = ['https://zenodo.org/record/6010450/files/']
+sources = ['DIRAC-%(version)s-Source.tar.gz']
+checksums = ['c8c011b5bc2ca36a2ef76b2deb911638321655fcae1d1b146cacfa1b5f783711']
+
+builddependencies = [('CMake', '3.20.1', '', ('GCCcore', '10.3.0'))]
+
+dependencies = [
+    ('Python', '3.9.5', '', ('GCCcore', '10.3.0')),
+    ('HDF5', '1.10.7', '', ('gompi', '2021a'))
+]
+
+separate_build_dir = True
+
+configopts = ' -DCMAKE_Fortran_COMPILER=mpif90 '
+configopts += '-DEXTRA_FCFLAGS="''" '
+configopts += '-DCMAKE_C_COMPILER=mpicc '
+configopts += '-DEXTRA_CFLAGS="''" '
+configopts += '-DCMAKE_CXX_COMPILER=mpicxx '
+configopts += '-DEXTRA_CXXFLAGS="''" '
+configopts += '-DENABLE_BLAS=auto '
+configopts += '-DENABLE_LAPACK=auto '
+configopts += '-DMKL_FLAG=off '
+configopts += '-DMATH_LIB_SEARCH_ORDER="MKL;ESSL;OPENBLAS;ATLAS;ACML;SYSTEM_NATIVE" '
+configopts += '-DBLAS_LANG=Fortran '
+configopts += '-DLAPACK_LANG=Fortran '
+configopts += '-DENABLE_MPI=True '
+configopts += '-DENABLE_OPENMP=False '
+configopts += '-DENABLE_CODE_COVERAGE=False '
+configopts += '-DENABLE_STATIC_LINKING=False '
+configopts += '-DENABLE_PROFILING=False '
+configopts += '-DENABLE_RUNTIMECHECK=False '
+configopts += '-DENABLE_64BIT_INTEGERS=False '
+configopts += '-DEXPLICIT_LIBS="off" '
+configopts += '-DENABLE_EXATENSOR=off '
+configopts += '-DENABLE_PCMSOLVER=off '
+configopts += '-G"Unix Makefiles" '
```

Comment:
Most of these should (haven't tried myself but...) not need explicit setting.
Most CMake files will correctly autodetect these things.
And make sure to not set things to their default value in case you are doing so here.

The one thing that may need fixing is the MATH_LIB stuff. The 2021a toolchain uses flexiblas (which has openblas as its backend). This may required a patch to DIRAC to make it figure that out. But the SEARCH_ORDER should not be set in here.